Integrations/integration-Centreon.yml

```diff
@@ -36,8 +36,7 @@ script:
     # disable insecure warnings
     requests.packages.urllib3.disable_warnings()
 
-    if not demisto.params().get('proxy', False):
-        disable_proxy()
+    handle_proxy()
```

Comment:
shouldn't we move this to the `EXECUTION CODE` section? 
i know until now we put it at the top, but is that really the best place?

also, this needs to be updated in the plugin template (after merging to master)

Reply:
Theoretically yes.

However, if we do that, any request that will be sent before the `EXECUTION CODE` section will try to pass through proxy. I know it shouldn't happen, but i've already seen integrations with active code before that block.

Reply:
In the case of this integration, yes, i'll do it now.